src/components/motion/MotionController.h

```diff
@@ -18,6 +18,16 @@ namespace Pinetime {
         BMA425,
       };
 
+      enum class Days : uint8_t {
+        Today = 0,
+        Yesterday,
+        Last,
+      };
+
+      static constexpr size_t stepHistorySize = static_cast<std::underlying_type_t<Days>>(Days::Last); // Store this many day's step counter
```

Comment:
Yeah, theoretically out of bounds. In practice the CircularBuffer handles it internally by doing modulo (`%`) operation on the size, so `Days::Last` would be just index 0 again.

While it **is** nice to refer to the days with `Today` and `Yesterday`, maybe having to name 5 more enum values is not that nice.

One option is that the `Days` enum can be removed altogether. Or alternatively it can remain with only `Today` and `Yesterday` values, and `NbSteps()` function could get an overload that expects a `uint8_t`.

Reply:
That second option sounds good to me, that way we have a nice human readable enum while also allowing specifying a number of days ago for more flexible usage